compiler/rustc_llvm/llvm-wrapper/PassWrapper.cpp

```diff
@@ -1205,7 +1205,11 @@ struct LLVMRustThinLTOData {
   // Not 100% sure what these are, but they impact what's internalized and
   // what's inlined across modules, I believe.
 #if LLVM_VERSION_GE(18, 0)
+#if LLVM_VERSION_GE(20, 0)
+  FunctionImporter::ImportListsTy ImportLists;
```

Comment:
I don't think you need this conditional. On older versions you can use the typedef as well.

Reply:
Looks like the typedef `FunctionImporter::ImportListsTy` is new as well: https://github.com/llvm/llvm-project/commit/4f15039cf2d59dbb889903aff8ac32ff266c8c48#diff-870a2d0fa51c9349d66da88fd1938cef621de9b4ac0c96d71d694b90dbefbfebR153.